platformio.ini

```diff
@@ -399,7 +402,7 @@ build_flags = ${common.build_flags_esp32}
   -D TEMPERATURE_PIN=23
 lib_deps = ${esp32.lib_deps}
   OneWire@~2.3.5
-  U8g2@~2.28.11
+  olikraus/U8g2 @ ^2.28.8
```

Comment:
v2.28.11 was not available via platformio.
Adding the fully-qualified library and requiring only v2.28.8 works.